src/main/java/uk/gov/companieshouse/pscdataapi/config/WebSecurityConfig.java

```diff
@@ -12,23 +14,31 @@
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
-
 import uk.gov.companieshouse.api.filter.CustomCorsFilter;
 import uk.gov.companieshouse.api.interceptor.InternalUserInterceptor;
 import uk.gov.companieshouse.api.interceptor.UserAuthenticationInterceptor;
 
 @Configuration
 public class WebSecurityConfig implements WebMvcConfigurer {
+    // feature flag marked for future removal
+    @Value("${feature.psc_individual_full_record_get:false}")
+    private Boolean featurePscIndividualFullRecordGet;
+// TODO: integration test
```

Comment:
Did this integration test get added? 
if so TODO can be removed

Reply:
Good spot, yes this is definitely covered by `CompanyPscFullRecordGetDisabledControllerTest` and Cucumber test `get_individual_full_record.feature`
I'll remove the comment.